aten/src/ATen/native/quantized/cpu/fbgemm_utils.cpp

```diff
@@ -197,3 +205,75 @@ Tensor ConvertToChannelsLast3dTensor(const Tensor& src) {
 } // namespace at
 
 #endif // USE_FBGEMM
+
+
+torch::jit::class_<LinearPackedParamsBase> register_linear_params() {
+  using SerializationType = std::tuple<at::Tensor, c10::optional<at::Tensor>, std::string, std::string>;
+  static auto register_linear_params =
+      torch::jit::class_<LinearPackedParamsBase>("LinearPackedParamsBase")
+          .def_pickle(
+              [](const c10::intrusive_ptr<LinearPackedParamsBase>& params)
+                  -> SerializationType { // __getstate__
+                at::Tensor weight;
+                c10::optional<at::Tensor> bias;
+                std::tie(weight, bias) = params->unpack();
+                return std::make_tuple(
+                    std::move(weight),
+                    std::move(bias),
+                    params->backend(),
```

Comment:
same comment as before: we should NOT serialize the backend and instead infer it from qengine setting at the time of loading